### DISCUSSION TITLE
why was auto driver enabled instead of driving a course to the sell point

### DISCUSSION THREAD

**Discussion by newride** [Courseplay/Courseplay_FS22#2995] [General]
Why was the ability to make a course to the sell point like in fs19 removed from CP?

**Comment by BalticHawk** [comment]
CP developer team decided to focus on field work and work together with [AutoDrive](https://github.com/Stephan-S/FS22_AutoDrive/releases/tag/2.0.1.4).
You can use this mod for creating a streetnetwork and tell CP to use this to get to refill and unload.

**Comment by Tensuko** [comment]
Nothing was removed, we just decided to not re-implement that feature.
Giants Helper can do this on some maps, like we wrote in out manual and as Tritura said, you can do this with AD.
